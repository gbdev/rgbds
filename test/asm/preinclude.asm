warn "main {__FILE__}"
def v3 = v1 + v2
println "{d:v1} + {d:v2} = {d:v3}"
