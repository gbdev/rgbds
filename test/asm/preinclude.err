warning: preinclude.asm(0) -> preinclude.inc(1): [-Wuser]
    pre-include "preinclude.inc"
warning: preinclude.asm(1): [-Wuser]
    main "preinclude.asm"
