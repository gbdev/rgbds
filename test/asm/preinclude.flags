-Weverything -P preinclude-1.inc -P preinclude-2.inc
