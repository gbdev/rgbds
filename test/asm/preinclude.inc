warn "pre-included file"

def v1 = 12
rept 3
	println "rept 3"
endr

def v2 = 34
for i, 3
	println "for {d:i}/3"
endr
