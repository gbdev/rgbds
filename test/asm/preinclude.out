rept 3
rept 3
rept 3
for 0/3
for 1/3
for 2/3
12 + 34 = 46
